README.md

```diff
@@ -1,14 +1,71 @@
-SQL Server Extension for Azure Functions
-===
+# Azure SQL Extension for Azure Functions #
 
-This repository contains extension code for SQL Server input and output bindings. Trigger bindings are coming soon.
+## Introduction ##
 
-# Input and Output Binding
-## Quick Start
+This repository contains extension code for the Azure SQL trigger and bindings as well as a quick start, tutorial, and samples of how to use them. A high level explanation of the trigger and bindings is provided below. Additional information for each is in their respective sample sections.
 
-### .NET Function App
+- **input binding**: takes a SQL query to run on a provided table and returns the output of the query.
+- **output binding**: takes a list of rows and upserts them into the user table (i.e. If a row doesn't already exist, it is added. If it does, it is updated).
+- **trigger**: requires the user to specify the name of a table, and in the event a change occurs (i.e. the row is updated, deleted, or inserted), the trigger will return the updated rows and values along with any associated metadata.
 
-1. Add MyGet package feed.
+## Table of Contents ##
+
+- [Quick Start](#Quick-Start)
+  - [SQL Setup](#SQL-Setup)
+  - [Set Up Local .NET Function App](#Set-Up-Local-.NET-Function-App)
+- [Tutorials](#Tutorials)
+  - [Create Azure SQL Database](#Create-Azure-SQL-Database)
+  - [Input Binding Tutorial](#Input-Binding-Tutorial)
+  - [Output Binding Tutorial](#Output-Binding-Tutorial)
+  - [Trigger Tutorial](#Trigger-Tutorial)
+- [Samples](#Samples)
+  - [Input Binding](#Input-Binding)
+  - [Output Binding](#Output-Binding)
+  - [Trigger](#Trigger)
+
+## Quick Start ##
+
+### SQL Setup ###
+
+This requires already having a SQL database. If you need to create a SQL database, please refer to [Create Azure SQL Database](#Create-Azure-SQL-Database) in the tutorials section.
+
+A primary key must be set in your SQL table before using the bindings. To do this, run the below SQL commands in the SQL query editor.
+
+1. Ensure there are no NULL values in the primary key column. The primary key will usually be an ID column.
+
+    ```sql
+    ALTER TABLE ['your table name'] alter column ['column to be primary key'] int NOT NULL
+    ```
+
+1. Set primary key column.
+
+    ```sql
+    ALTER TABLE ['your table name'] ADD CONSTRAINT PKey PRIMARY KEY CLUSTERED (['column to be primary key']);
+    ```
+
+1. Change tracking must be enabled on the database to use the trigger. If you do not plan on using the trigger, you can skip this step. To enable change tracking on the database, run:
```

Comment:
I would include an FYI that Change Tracking costs money :) (or a link to their overview docs)

Reply:
Good point. I didn't know this was actually the case. I'll update this